pom.xml

```diff
@@ -88,36 +93,57 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<!-- JSON parsing -->
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-core</artifactId>
 			<version>${ver.jackson}</version>
 		</dependency>
+		<!-- Jena, turtle -->
 		<dependency>
 			<groupId>org.apache.jena</groupId>
-			<artifactId>jena-osgi</artifactId>
+			<artifactId>jena-core</artifactId>
 			<version>${jena.version}</version>
 		</dependency>
 		<dependency>
 			<groupId>com.github.jsonld-java</groupId>
 			<artifactId>jsonld-java</artifactId>
 			<version>${ver.jsonldjava}</version>
 		</dependency>
+		<!-- Git in Java -->
 		<dependency>
 			<groupId>org.eclipse.jgit</groupId>
 			<artifactId>org.eclipse.jgit</artifactId>
 			<version>5.13.0.202109080827-r</version>
```

Comment:
Bumping this to 6.x breaks the build. I think jgit 6.x requires Java 11, but our Docker+Docker Compose image didn't work when I tested upgrading it. For a follow-up/later.

Reply:
Can you make an issue for this?

Reply:
Good idea, done! https://github.com/common-workflow-language/cwlviewer/issues/367